federatedrec/transfer_variable/definition/homo_fm.json

```diff
@@ -0,0 +1,141 @@
+{
+  "HomoFMTransferVariable":{
+    "guest_uuid": {
+      "src": "guest",
+      "dst": [
+        "arbiter"
+      ]
+    },
+    "host_uuid": {
+      "src": "host",
+      "dst": [
+        "arbiter"
+      ]
+    },
+    "uuid_conflict_flag": {
+      "src": "arbiter",
+      "dst": [
+        "guest",
+        "host"
+      ]
+    },
+    "dh_pubkey": {
+      "src": "arbiter",
+      "dst": [
+        "guest",
+        "host"
+      ]
+    },
+    "dh_ciphertext_host": {
+      "src": "host",
+      "dst": [
+        "arbiter"
+      ]
+    },
+    "dh_ciphertext_guest": {
+      "src": "guest",
+      "dst": [
+        "arbiter"
+      ]
+    },
+    "dh_ciphertext_bc": {
+      "src": "arbiter",
+      "dst": [
+        "guest",
+        "host"
+      ]
+    },
+    "paillier_pubkey": {
```

Comment:
If Paillier encryption is no needed, we suggest to remove transfer_variable relative with it to avoid potential misunderstanding.